.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -22,22 +22,23 @@ jobs:
           python-version: '2.x'
           architecture: 'x64'
       - name: Setup python by installing wheel
-        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: pip install wheel==0.35.0
       - name: Install third party
-        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: python -m scripts.install_third_party_libs
+      - id: check-risk
```

Comment:
[Optional] To avoid duplicate if checks can we use `check-work` structure as suggested [here](https://stackoverflow.com/a/61832535)?

Reply:
That's essentially what we're doing. The linked solution sets `needs: <step with the check>` for every step. Then those steps will only run if the output of `step with the check` was truthy. The current code does the same thing, just with different syntax. Every job sets `if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}` so that it runs only if the `is-low-risk` output of the `check-risk` step (equivalent to `step with the check` before) is nonzero (equivalent to truthy before).

Reply:
I was suggesting the last solution mentioned in the answer:
```
jobs:
  check:
    steps:
      - id: ...
        name: Install python-libs
        run: ...
      - id: ...
        name: check-if-pr-is-low-risk
        run: ...
  work:
    needs: check
    if: not low-risk-pr
    steps:
      - run: install-other-libs
      - run: e2e-test
```
This^ way we just have to do the check once right? i.e, not in every step.

Reply:
Yes, I think that will work. I think this is pretty low priority at the moment though. We've mostly been postponing these kinds of "nice to have" fixes until the flakiness is under control. I've opened #12647 to track this though